lite/Dockerfile

```diff
@@ -52,16 +52,18 @@ RUN git clone "$GIT_SERVER/ledger-secure-sdk.git" "$LEDGER_SECURE_SDK"
 ENV NANOS_SDK=/opt/nanos-secure-sdk
 RUN git clone --branch 2.1.0 --depth 1 "$GIT_SERVER/nanos-secure-sdk.git" "$NANOS_SDK"
 
-# Latest Nano X SDK (OS nanox_2.1.0)
+# Latest Nano X SDK (OS nanox_2.1.0 => based on API_LEVEL 1 => clone on SDK tag v1.2.0)
 ENV NANOX_SDK=/opt/nanox-secure-sdk
-RUN git -C "$LEDGER_SECURE_SDK" worktree add "$NANOX_SDK" v1.1.0
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$NANOX_SDK" v1.2.0
 RUN echo nanox > $NANOX_SDK/.target
 
-# Latest Nano S+ SDK (OS nanos+_1.1.0)
+# Latest Nano S+ SDK (OS nanos+_1.1.0 => based on API_LEVEL 1 => clone on SDK tag v1.2.0)
 ENV NANOSP_SDK=/opt/nanosplus-secure-sdk
-RUN git -C "$LEDGER_SECURE_SDK" worktree add "$NANOSP_SDK" v1.1.0
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$NANOSP_SDK" v1.2.0
 RUN echo nanos2 > $NANOSP_SDK/.target
```

Comment:
Not related to this PR in particular, but could we find a way to factorize the common parts of the `lite` and `legacy` dockerfiles ?

Reply:
From what I'm reading using:
https://docs.docker.com/build/building/multi-stage/
Would solve the need to duplicate all the SDK checkout

Reply:
I gave it a quick look and I'm not really sure how to do that? Anyway this can be done in a later PR if we don't have an easy way of using it. @apaillier-ledger maybe you know how this could be used?

Reply:
My last commit attempts to improve this.

Reply:
Good for me